sale_order_import/tests/test_sale_order_import.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import base64
+import os
+
+from odoo.tests import common
+from odoo.tools.misc import file_open
+from odoo.tools.safe_eval import safe_eval
+
+
+class TestSaleOrderImport(common.TransactionCase):
+    """
+    This tests allows to import the test
+    sale order csv file and import sale order.
+    """
+
+    def setUp(self):
+        super(TestSaleOrderImport, self).setUp()
+        self.import_sale_obj = self.env["import.sale"]
+
+        # Create Test Partner
+        self.partner_01 = self.env["res.partner"].create(
+            {"name": "Test Partner 01", "phone": "1234567890", "mobile": "1234567890"}
+        )
+        self.partner_02 = self.env["res.partner"].create(
+            {"name": "Test Partner 02", "phone": "9876543210", "mobile": "9876543210"}
+        )
+
+        # Create Test Product
+        self.product_01 = self.env["product.product"].create(
+            {"name": "Product A", "type": "product", "default_code": "ABC"}
+        )
+        self.product_02 = self.env["product.product"].create(
+            {"name": "Product B", "type": "product", "default_code": "XYZ"}
+        )
+        self.product_03 = self.env["product.product"].create(
+            {"name": "Product C", "type": "product", "default_code": "ZYX"}
+        )
+
+        self.public_pricelist = self.env.ref("product.list0")
+        self.warehouse = self.env.ref("stock.warehouse0")
+        self.team = self.env.ref("sales_team.crm_team_1")
+        self.free_delivery = self.env["ir.model.data"].xmlid_to_res_id(
+            "delivery.free_delivery_carrier"
+        )
+        self.customer_phone = "123456789"
+        self.cash_journal_cash = self.env["account.journal"].create(
+            {"name": "Cash", "type": "cash", "code": "CASH"}
+        )
+        self.note = "Test Note"
+
+        # Check the file path and load the test_sale_order.csv
+        self.file_path = os.path.join(
+            "sale_order_import", "tests", "test_sale_order.csv"
+        )
+        self.generated_file = file_open(self.file_path, "rb")
+        self.generated_file = self.generated_file.read()
+
+    def test_get_order_dict_flow_01(self):
```

Comment:
Wouldn't `test_01_get_order_dict_flow` make a better naming convention because the positions of the numbers get inconsistent when the tests are listed (e.g. in the traceback of the test result)?  @TimLai125 Please comment.

Reply:
Yea, but so far we do not have a standard way on naimg the test method, @YogeshMahera-SerpentCS Let's try to have the following naming logic:
1. test_01_
2. test_02_